db/migrate/20220131162210_add_oauth_parameters_to_users.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Added Oauth uid and provider to users.
+class AddOauthParametersToUsers < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured { add_column :users, :uid, :string }
+    safety_assured { add_column :users, :provider, :string }
+    safety_assured { remove_column :users, :password_digest, :string }
+    safety_assured { remove_column :users, :perishable_token, :string }
```

Comment:
perishable_token is how tests logged in without having to do the full login flow. I wonder how we can replicate that. 🤔

Reply:
That's the issue I am having. I'll keep perishable token.